R/pkg/R/DataFrame.R

```diff
@@ -3905,6 +3905,16 @@ setMethod("rollup",
             groupedData(sgd)
           })
 
+isTypeAllowedForSqlHint <- function(x) {
+  if (is.character(x) | is.numeric(x)) {
```

Comment:
nit: very minor, both is.* returns a single element, but with `||` you get shortcut eval if is.character(x) is TRUE